test/offsetarray.jl

```diff
@@ -313,7 +313,7 @@ seek(io, 0)
 @test readdlm(io, eltype(A)) == parent(A)
 
 amin, amax = extrema(parent(A))
-@test clamp(A, (amax+amin)/2, amax) == clamp(parent(A), (amax+amin)/2, amax)
+@test clamp.(A, (amax+amin)/2, amax).parent == clamp.(parent(A), (amax+amin)/2, amax)
```

Comment:
The operation on the left produces a `TestHelpers.OAs.OffsetArray` (from `test/TestHelpers.jl`) whereas that on the right produces an `Array`. Comparing an `OffsetArray` to an `Array` seems to fail even when their contents are identical, hence the `.parent`. Thoughts? Thanks!